dpxdt/client/workers.py

```diff
@@ -67,13 +71,22 @@ def _print_tree(obj):
     def _get_dict_for_repr(self):
         return self.__dict__
 
-    def __repr__(self):
+    def _print_repr(self, depth):
+        if depth <= 0:
+            return '%s.%s#%d' % (
+                self.__class__.__module__,
+                self.__class__.__name__,
+                id(self))
+        
         return '%s.%s(%s)#%d' % (
             self.__class__.__module__,
             self.__class__.__name__,
-            self._print_tree(self._get_dict_for_repr()),
+            self._print_tree(self._get_dict_for_repr(), depth - 1),
```

Comment:
good point, added a comment!

hmmm...maybe your client is doubling the spacing? it seems well under 80 chars